src/models/group.js

```diff
@@ -1,19 +1,25 @@
 import mongoose from 'mongoose';
-import User from './user';
+import uuid from 'uuid';
 
 const modelName = 'Group';
 
 const groupSchema = new mongoose.Schema({
   inviteCode: { type: String, required: true, unique: true },
-  users: [User.schema],
-  creator: { type: User.schema, required: true },
-  invites: [User.schema],
+  users: [mongoose.Schema.Types.ObjectId],
```

Comment:
```suggestion
  users: [{ type: mongoose.Schema.Types.ObjectId, ref: 'User' }],
```

If we want to use references to other objects we need to tell mongoose which collection this references and how to populate it.

Reply:
I read online that you can instead use the actual model as the ref instead of just its name.  Although its benefits will probably not be used in our case, I still think it is stronger.

Reply:
> I read online that you can instead use the actual model as the ref instead of just its name. Although its benefits will probably not be used in our case, I still think it is stronger.

If that works I'm fine with that